pkg/task/fetch.go

```diff
@@ -65,30 +65,23 @@ func NewK8sTaskSpecFetcher(searchNamespace string) (*K8sTaskSpecFetcher, error)
 //
 // NOTE:
 //  This is an implementation of TaskSpecFetcher interface
-func (f *K8sTaskSpecFetcher) Fetch(taskName string) (metaTaskYml string, taskYml string, err error) {
+func (f *K8sTaskSpecFetcher) Fetch(taskName string) (runtask RunTask, err error) {
 	if len(taskName) == 0 {
-		return "", "", fmt.Errorf("Nil task name: Task can not be fetched")
+		err = fmt.Errorf("failed to fetch runtask: nil task name was provided")
+		return
 	}
 
 	cm, err := f.k8sClient.GetConfigMap(taskName, mach_apis_meta_v1.GetOptions{})
```

Comment:
Is it still configmap? Is it not changed to custom resource?

Reply:
No its not changed yet.